turkce/catalla-kaz/metali-cek/iple-in/yukari-cik/yukari-cik.md

```diff
@@ -0,0 +1,8 @@
+İple atlamayı yeterince mantıklı bulmadığından geriye koşup diğer yoldan gitmeye karar verdin. 
+
+Hızla geriye doğru koşmaya başladın fakat hatmiçieği çoktan diğer yola olan dönüşü geçti.
+
+[İple inebileceğin karanlığa geri dön](../../asagi-atla/asagi-atla.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.